meshroom/videoUtils/ConvertVideoToSeq.py

```diff
@@ -0,0 +1,57 @@
+__version__ = "1.0"
+
+import os
+import logging
+from pathlib import Path
+from meshroom.core import desc
+
+
+class ConvertVideoToSeq(desc.CommandLineNode):
+    commandLine = "ffmpeg -y -i {videoFileValue}"
+    gpu = desc.Level.NONE
+
+    category = "Video Utils"
+    documentation = """
+Extract frames from a video.
+"""
+
+    inputs = [
+        desc.File(
+            name="videoFile",
+            label="Video File",
+            description="Video file to extract the frames from.",
+            value="",
+        ),
+        desc.StringParam(
+            name="pattern",
+            label="Output Sequence Pattern",
+            description="Filename pattern for the output frames.",
+            value="frame_%04d.png",
+        ),
+    ]
+
+    outputs=[
+        desc.File(
+            name="outputFolder",
+            label="Output Folder",
+            description="Folder where to save the frames.",
+            value=lambda attr: os.path.join("{nodeCacheFolder}", "frames"),
+        ),
```

Comment:
It would be good to add:
```suggestion
        ),
        desc.File(
            name="outputSeq",
            label="Image Sequence",
            semantic="sequence",
            description="Output image sequence.",
            value="{nodeCacheFolder}/{patternValue}",
            group="",
        ),
```